cub/cub/util_namespace.cuh

```diff
@@ -102,13 +102,14 @@
 #  define CUB_DETAIL_MAGIC_NS_END
 #else // not defined(CUB_DISABLE_NAMESPACE_MAGIC)
 #  if defined(_NVHPC_CUDA)
-#    define CUB_DETAIL_MAGIC_NS_BEGIN                                                                  \
-      inline namespace _CCCL_PP_SPLICE_WITH(_, CUB, CUB_VERSION, SM, NV_TARGET_SM_INTEGER_LIST, NVHPC) \
+#    define CUB_DETAIL_MAGIC_NS_BEGIN                                                                                  \
+      inline namespace _CCCL_PP_CAT(                                                                                   \
+        _CCCL_PP_CAT(_CCCL_PP_CAT(_V_, CUB_VERSION), _CCCL_PP_SPLICE_WITH(_, _SM, NV_TARGET_SM_INTEGER_LIST)), _NVHPC) \
```

Comment:
I would say the "simpler" fix would be to just use `_CUB` and not `CUB`

That wont change the actual string in any recognizable way and is a single character change. 

Note we should also change the CCCL macro in `namespace.h`

Reply:
Oh we can happily add `_` there too